app/controllers/api/audio_files_controller.rb

```diff
@@ -7,17 +7,17 @@ class Api::AudioFilesController < Api::BaseController
   filter_resources_by :audio_version_id
 
   # if the audio version is not set, and story is, set default av
-  def resource
+  def create_resource
     super.tap do |af|
-      if story && af.try(:new_record?)
+      if story
         af.audio_version_id ||= story.default_audio_version.id
         af.account_id ||= story.account_id
```

Comment:
The rest of this is actually no longer necessary, it is handled by `resources_base` defined as below, so if we update `Story` to work that way we can get rid of this. I didn't want to overload the pull request.